ruff.toml

```diff
@@ -1,6 +1,7 @@
 [lint]
 extend-select = [
 	"C901",
+	"RUF100",
```

Comment:
I'm not sure if I'm prepared to accept the toil that will come from merging conflicts in this file. I'm happy to apply the changes but would like to defer enforcing them until the skeleton can accept the rule.